drivers/wifi/nrfwifi/Kconfig.nrfwifi

```diff
@@ -700,4 +700,11 @@ config NRF_WIFI_QOS_NULL_BASED_RETRIEVAL
 
 endchoice
 
+config NRF_WIFI_BUILD_ONLY_MODE
+	bool "Build only mode"
```

Comment:
I just wonder if there is an option that is enabled when CI is being run, then we could depend on it here and things would work automatically.

Reply:
I thought the same too, but couldn't find any thing. At most we can do is have a ENV variable, but not sure if we can use that in the Kconfig.